process/dataValidators/txValidator.go

```diff
@@ -76,14 +77,27 @@ func (txv *txValidator) CheckTxValidity(interceptedTx process.InterceptedTransac
 		return nil
 	}
 
+	// for relayed v3, we allow sender accounts that do not exist
+	isRelayedV3 := common.IsRelayedTxV3(interceptedTx.Transaction())
+	hasValue := hasTxValue(interceptedTx)
+	shouldAllowMissingSenderAccount := isRelayedV3 && !hasValue
 	accountHandler, err := txv.getSenderAccount(interceptedTx)
-	if err != nil {
+	if err != nil && !shouldAllowMissingSenderAccount {
 		return err
 	}
 
 	return txv.checkAccount(interceptedTx, accountHandler)
 }
 
+func hasTxValue(interceptedTx process.InterceptedTransactionHandler) bool {
```

Comment:
Indeed, we don't handle `MultiESDTTransfer` with EGLD in interceptor (we never did, but that is OK).